app/services/email_output_service.rb

```diff
@@ -0,0 +1,70 @@
+class EmailOutputService
+  def initialize(email_service:)
```

Comment:
Is email_service an adapter / gateway?

Reply:
this projects uses the word service. I take that to meen usecase

Reply:
Right I see.
Responsibilities of these objects seem a bit blurred to me.
Why does EmailService functionality not get absorbed into EmailOutputService or vice versa?

Reply:
I agree, EmailService is an exsisitng service that should be merged into this one in a later stage. This class is dark at the moment

Reply:
I see `email_output_service` as the UseCase and `email_service` as the Gateway.